tracking/src/main/java/org/motechproject/nms/tracking/service/CacheService.java

```diff
@@ -0,0 +1,5 @@
+package org.motechproject.nms.tracking.service;
+
+public interface CacheService {
```

Comment:
Or if we keep the new service, instead of calling it CacheService I think we should rename it the EntityServiceWrapper and not expose it.  So instead of placing it in the service package we should either see if we've done something like this before and copy that, or create a new package internal/service so it isn't exposed via osgi.